components/Python/Scikit-learn/dataframe_to_file/dataframe_to_file.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import print_function
+
+import argparse
+import sys
+import time
+import os
+import pandas
+
+from parallelm.components import ConnectableComponent
+
+class MCenterWriteDFComponentAdapter(ConnectableComponent):
+    """
+    Adapter for write_df_to_file 
+    """
+
+    def __init__(self, engine):
+        super(self.__class__, self).__init__(engine)
+
+    def _materialize(self, parent_data_objs, user_data):
+        file_path = self._params["sinkfilename"]
+        df_data = parent_data_objs[0]
+        return[write_df_to_file(df_data, file_path)]
+
+
+def write_df_to_file(df_data, filepath):
+    """
+    Save DataFrame to file
+    """
+    suffix_time_stamp = str(int(time.time()))
+    save_file = str(filepath) + '.' + suffix_time_stamp
+    sfile = open(save_file, 'w+')
+    pandas.DataFrame(df_data).to_csv(save_file)
```

Comment:
According to docs, first param of the to_csv func is:
`path_or_buf : str or file handle, default None
File path or object, if None is provided the result is returned as a string. If a file object is passed it should be opened with newline=’‘, disabling universal newlines.

Changed in version 0.24.0: Was previously named “path” for Series.`

you open a file, but pass a path.... 
As I undestand, you can to pass path; or open and pass the object. Check please.

Reply:
$ pip freeze | grep pandas
pandas==0.24.2

parameter to "to_csv" is string so it is valid, however I will add "path_or_buf=" so as its clear specifically for the release post 0.24.0